### ISSUE TITLE
proc.ListProcesses is shared but uses instance fields

### ISSUE THREAD

**Issue by lstefano71** [Dyalog/library-core#1]
Try the following:
```
      APLProcess.ListProcesses ''
SYNTAX ERROR: Access error: Cannot access instance field from within a class
ListProcesses[11] :If IsWin
                 ∧
      
```
Possibly caused by commit https://github.com/Dyalog/library-core/commit/38b1d9e37e4a29890f3f74b7db2b29ee5ce94686?

**Comment by lstefano71** [comment]
Proposed workaround: https://github.com/Dyalog/library-core/pull/2

**Comment by bpbecker** [comment]
Fixed by 9969bdcfd8a695662cdc6a02f233e0495c95e014
